subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt

```diff
@@ -145,22 +134,36 @@ fun provideEnvironment(
                     null
                 )[0]
             },
-            // from GradleInstrumentationPluginConfiguration
-            slackToken = argumentsProvider.getMandatoryArgument("slackToken"),
             elasticConfig = ElasticConfigFactory.parse(argumentsProvider),
             sentryConfig = parseSentryConfig(argumentsProvider),
             statsDConfig = parseStatsDConfig(argumentsProvider),
             fileStorageUrl = argumentsProvider.getMandatoryArgument("fileStorageUrl"),
             testRunCoordinates = coordinates,
-            reportConfig = reportConfig,
-            isSyntheticStepsEnabled =
-            argumentsProvider.getOptionalArgument("isSyntheticStepsEnabled")?.toBoolean() ?: false
+            reportDestination = parseReportDestination(argumentsProvider)
         )
     } catch (e: Throwable) {
         TestRunEnvironment.InitError(e.message ?: "Can't parse arguments for creating TestRunEnvironment")
     }
 }
 
+private fun parseReportDestination(argumentsProvider: ArgsProvider): ReportDestination {
+    val buildId = argumentsProvider.getMandatoryArgument("teamcityBuildId").toInt()
+    return if (buildId == TestRunEnvironment.LOCAL_STUDIO_RUN_ID) {
+        val isReportEnabled = argumentsProvider.getOptionalArgument("avito.report.enabled")?.toBoolean() ?: false
```

Comment:
I expect to disable any transport with `avito.report.enabled=false`, not only backend

Reply:
I agree with you, It's a problem with a property name
I don't want to change semantics in that PR